### PR TITLE
net-vpn/eduvpn-client: new package

### PR DESCRIPTION
**Pull request by hamarituc**
This PR adds the eduVPN client used by several European academic institutions to feature an easy to use VPN configuration utility.

My employer implements this VPN solution in the future so I hosted the necessary Go dependencies on my personal web space of my institution. Please let me know if an other download source would be more appropriate.

### CODE REVIEW

File: net-vpn/eduvpn-client/eduvpn-client-4.1.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	SRC_URI="https://github.com/eduvpn/python-eduvpn-client/archive/refs/tags/${PV}.tar.gz -> ${P}.gh.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/python-${P}"
+fi
+
+DESCRIPTION="Linux client and Python client API for eduVPN"
+HOMEPAGE="https://www.eduvpn.org/"
+
+SLOT="0"
+LICENSE="GPL-3+"
```
Comment (ulm) [inline net-vpn/eduvpn-client/eduvpn-client-4.1.3.ebuild:25]: Please use canonical ordering of variables.
```suggestion
LICENSE="GPL-3+"
SLOT="0"
```

File: net-vpn/eduvpn-client/eduvpn-client-4.1.99.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	SRC_URI="https://github.com/eduvpn/python-eduvpn-client/archive/refs/tags/${PV}.tar.gz -> ${P}.gh.tar.gz"
+	S="${WORKDIR}/python-${P}"
+fi
+
+DESCRIPTION="Linux client and Python client API for eduVPN"
+HOMEPAGE="https://www.eduvpn.org/"
+
+SLOT="0"
+LICENSE="GPL-3+"
+
```
Comment (ulm) [inline net-vpn/eduvpn-client/eduvpn-client-4.1.99.0.ebuild:29]: Canonical ordering, no blank line between `SLOT` and `RESTRICT`.
```suggestion
LICENSE="GPL-3+"
SLOT="0"
```

File: net-vpn/eduvpn-common/eduvpn-common-1.1.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+		https://github.com/eduvpn/eduvpn-common/archive/refs/tags/${PV}.tar.gz -> ${P}.gh.tar.gz
+		https://www-user.tu-chemnitz.de/~hamari/eduvpn/${P}-deps.tar.xz
+	"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="openvpn wireguard"
+
```
Comment (ulm) [inline net-vpn/eduvpn-common/eduvpn-common-1.1.2.ebuild:29]: No blank line here.

File: net-vpn/eduvpn-common/eduvpn-common-1.1.99.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+	SRC_URI="
+		https://github.com/eduvpn/eduvpn-common/archive/refs/tags/${PV}.tar.gz -> ${P}.gh.tar.gz
+		https://www-user.tu-chemnitz.de/~hamari/eduvpn/${P}-deps.tar.xz
+	"
+fi
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="openvpn wireguard"
+
```
Comment (ulm) [inline net-vpn/eduvpn-common/eduvpn-common-1.1.99.0.ebuild:28]: Ditto.

File: net-vpn/eduvpn-client/eduvpn-client-4.1.3.ebuild (hunk 1 above)
Comment (hamarituc) [reply net-vpn/eduvpn-client/eduvpn-client-4.1.3.ebuild]: fixed

File: net-vpn/eduvpn-client/eduvpn-client-4.1.99.0.ebuild (hunk 2 above)
Comment (hamarituc) [reply net-vpn/eduvpn-client/eduvpn-client-4.1.99.0.ebuild]: fixed

File: net-vpn/eduvpn-common/eduvpn-common-1.1.99.0.ebuild (hunk 4 above)
Comment (hamarituc) [reply net-vpn/eduvpn-common/eduvpn-common-1.1.99.0.ebuild]: fixed

**Comment by ulm** [comment]
Merged. :smile:

**Pull request closed without merging**